src/app/projects/add-edit-project/add-edit-project.component.html

```diff
@@ -170,19 +182,64 @@ <h2>Location</h2>
               </li>
             </ul>
           </div>
-        </div>
-	    <div class="small-margin-bottom half-wide">
-          <label for="shapeFileColour">Shape File Colour</label><br>
-		  <input 
-            class="shape-file-colour-input"
-            formControlName="shapeFileColour"
-            [colorPicker]="myForm.controls.shapeFileColour.value"
-            (colorPicker)="myForm.controls.shapeFileColour.setValue($event)"
-            [style.background]="myForm.controls.shapeFileColour.value"
-            [style.color]="shapeFileTextColour"
-            (colorPickerChange)="updateShapeFileColour($event)"
-        />
-        </div>
+        </div> -->
+
+      <div class="logos-buttons flex-container half-wide">
+        <input
+          type="button"
+          id="shapefiles"
+          class="btn btn-primary small-margin-bottom logos-button"
+          value="{{shapefiles?.length > 0 ? 'Reselect shapefiles(s)' : 'Select shapefiles(s)'}}"
+          (click)="launchFilePicker('shapefiles', 'Select project shapefiles(s)', false, 'zip', ['application/zip'], 6)"
+        >
+        <ul class="shapefile-list mb-3" *ngIf="shapefiles?.controls.length > 0">
+          <li class="shapefile-list-item" *ngFor="let shapefile of shapefiles?.controls; index as i">
+            <div class="shapefile-list-item-segment">
+              <span class="" [title]="">
+                <span class="">{{shapefile.controls.documentFileName.value}}</span>
```

Comment:
Will this value always be available?

Reply:
I'll double-check this to see if it'll be safe

Reply:
There's a conditional above so we should be good here!